### DISCUSSION TITLE
cPanel Repo Clone Issues

### DISCUSSION THREAD

**Discussion by JMakiol** [community/community#38276] [Other Feature Feedback, Questions, & Ideas]
Hi everyone,

I'm creating a new website and would like to manage my files through github, but I'm having some issues. I've created and linked the SSH key and my cPanel terminal says I've successfully authenticated when I use the <ssh -i ~/.ssh/##repo_name## -T git@github.com> command. However, I can't manage to clone the repo to my Github account and I've enhausted what I can find online and on the forums.

Any idea what I'm doing wrong? Any help is appreciated.

Thanks!
James

**Comment by airtower-luna** [comment]
> However, I can't manage to clone the repo to my Github account and I've enhausted what I can find online and on the forums.

What exactly are you trying to do? In Git terms, "cloning" refers to the operation that creates a fresh copy of a remote repository on your local machine.

**Reply by JMakiol** [reply]
I'm trying to make a copy of my cPanel repository in my GitHub Desktop app to use on my local machine

**Comment by david-engelmann** [comment]
I've had this exact issue yesterday, I needed to update my .git/config to point to the correct url. [Check out this issue for more details](https://github.com/tpope/vim-fugitive/issues/2074). Special shoutout to tpope!
